script/test_todo.py

```diff
@@ -0,0 +1,52 @@
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+
+
+def add_task(driver, title):
+  input_todo = driver.find_element_by_xpath("//html/body/section/div/header/input")
+  input_todo.clear()
+  input_todo.send_keys(title)
+  input_todo.send_keys(Keys.RETURN)
+
+
+def verify_active_task(driver, id):
+  elem = driver.find_element_by_xpath(f"/html/body/section/div/section/ul/li[{id}]")
+  class_elem = elem.get_attribute("class")
+  assert class_elem != "completed"
+
+
+def verify_tasks_different(driver, id1, id2):
+  elem_1 = driver.find_element_by_xpath(f"/html/body/section/div/section/ul/li[{id1}]")
+  elem_2 = driver.find_element_by_xpath(f"/html/body/section/div/section/ul/li[{id2}]")
+  assert elem_1.text != elem_2.text  
+
+
+def click_task(driver, id):  
+  elem_toggle = driver.find_element_by_xpath(f"/html/body/section/div/section/ul/li[{id}]/div/input")
+  elem_toggle.click()
+
+
+def verify_task_completed(driver, id):
+  elem = driver.find_element_by_xpath(f"/html/body/section/div/section/ul/li[{id}]")
+  assert elem.get_attribute("class") == "completed"
+
+
+
+def main():
```

Comment:
It looks ok. Just last one change which will help us in the future.

Rename function `main` into `test_task_completed` and then create new `main` function witch will only call this `test_task_completed` function. So every scenario will have it own function and those function will be called from main one by one.